lms/templates/basic_lti_launch/canvas_file_basic_lti_launch.html.jinja2

```diff
@@ -1,17 +1 @@
-{% extends "lms:templates/basic_lti_launch/basic_lti_launch.html.jinja2" %}
-
-{% block content %}
-  {% if via_url %}
-    {{ super() }}
-  {% else %}
-    <main class="modal-content">
-      <h1>Authorize Hypothesis</h1>
-
-      <p class="modal-text">
-        Hypothesis needs your authorization to access this assignment.
-      </p>
-
-      <button class="btn" type="submit">Authorize</button>
-  </main>
-  {% endif %}
-{% endblock %}
```

Comment:
If the frontend receives an error from the `via_url` API then it will have to show a dialog with an <kbd>Authorize</btn> like this one. (After the user closes the authorization window the frontend will re-try the `via_url` request and if the request fails again the frontend will return to this dialog with the <kbd>Authorize</btn> button again.)